hack/build-manifests.sh

```diff
@@ -325,7 +326,8 @@ ${PROJECT_ROOT}/tools/manifest-templator/manifest-templator \
   --aaq-version="${AAQ_VERSION}" \
   --operator-image="${HCO_OPERATOR_IMAGE}" \
   --webhook-image="${HCO_WEBHOOK_IMAGE}" \
-  --cli-downloads-image="${HCO_DOWNLOADS_IMAGE}"
+  --cli-downloads-image="${HCO_DOWNLOADS_IMAGE}" \
+  --primary-udn-image="${}"
```

Comment:
you need use the `PRIMARY_USER_DEFINED_NETWORK_BINDING_IMAGE` var here

Reply:
Also, the parameter name is wrong: should be `primary-udn-binding-image`

Reply:
oh damn, I though I got everything.
Thanks for the catch!